pandasai/smart_dataframe/__init__.py

```diff
@@ -65,10 +65,14 @@ def __init__(
 
         self._load_df(df)
 
+        self._load_config(config)
```

Comment:
Wouldn't save _config in the smart df, since it always relies on the one that we store on the smart dl.